bin/system-info

```diff
@@ -120,7 +124,15 @@ fi
 PRIMARY_VRAM=$(grep "Dedicated video memory:" /tmp/glxinfo.txt | cut -d':' -f2 | sed 's/^ *//')
 
 # Sound card:
-AUDIO_DEVICE=$(pulsemixer --list-sinks | grep Default | cut -d',' -f2 | sed 's/ Name: //')
+if icanuse pulsemixer; then
+    AUDIO_DEVICE=$(pulsemixer --list-sinks | grep Default | cut -d',' -f2 | sed 's/ Name: //')
+elif icanuse pw-dump && icanuse pactl && icanuse jq; then
+    AUDIO_DEVICE_ALSA=$(pactl get-default-sink)
+    AUDIO_DEVICE_QUERY=".[] | select(.type == \"PipeWire:Interface:Node\") | select(.info.props.\"node.name\" == \"$(pactl get-default-sink)\") | .info.props.\"node.nick\""
```

Comment:
This does not work on my machine. There does not appear to be any matching sink in the output of `pw-dump`.

Reply:
Interesting.  What's your `pw-dump --version`?  0.3.65 produces what appears to be valid JSON (or at least valid enough for `jq` to make sense of it); maybe the format changed?

Reply:
```
$ pw-dump --version
pw-dump
Compiled with libpipewire 0.3.66
Linked with libpipewire 0.3.66
```

I think it just might be something with this particular hardware and/or a bug with pw-dump.
On another machine it worked fine. :shrug: